### PR TITLE
Clearer message on why PR validation failed

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by johanandren** [comment]
PLS BUILD

File: project/ValidatePullRequest.scala [hunk 1]
```diff
@@ -460,7 +460,9 @@ object AggregatePRValidation extends AutoPlugin {
       log.info(s"Wrote PR validation report to ${outputFile.getAbsolutePath}")
       //write(s"Overall result was: $result")
 
-      if (failed.nonEmpty || mimaFailures.nonEmpty || failedTasks.nonEmpty) throw new RuntimeException("Pull request validation failed!")
+      if (failed.nonEmpty) throw new RuntimeException(s"Pull request validation failed! Tests failed: $failed")
+      else if (mimaFailures.nonEmpty) throw new RuntimeException(s"Pull request validation failed! Mima failures: $mimaFailures")
+      else if (failedTasks.nonEmpty) throw new RuntimeException(s"Pull request validation failed! Failed tasks: $failedTasks")
```
Comment (Philippus) [inline project/ValidatePullRequest.scala]: Do these failure types exclude each other?

Comment (johanandren) [reply project/ValidatePullRequest.scala]: I think they are run in sequence

Comment (raboof) [reply project/ValidatePullRequest.scala]: I guess a failure could now obscure a MiMa error - let's leave that for further improvements like #865 though.

**Pull request merged**